api/db-cdr/csv/visit_criteria.csv

```diff
@@ -1,82 +1,82 @@
-316441,316402,VISIT,,,Inpatient Critical Care Facility,0,1,,581379,Visit,0,
-316442,316403,VISIT,,,Inpatient Critical Care Facility,0,1,,581379,Visit,0,
-316443,316402,VISIT,,,Inpatient Hospital,0,1,,8717,Visit,0,
-316444,316402,VISIT,,,Inpatient Intensive Care Facility,0,1,,581382,Visit,0,
-316445,316403,VISIT,,,Inpatient Intensive Care Facility,0,1,,581382,Visit,0,
-316446,316402,VISIT,,,Inpatient Long-term Care,0,1,,8970,Visit,0,
-316447,316404,VISIT,,,Inpatient Long-term Care,0,1,,8970,Visit,0,
-316448,316402,VISIT,,,Inpatient Nursery,0,1,,581384,Visit,0,
-316449,316402,VISIT,,,Inpatient Psychiatric Facility,0,1,,8971,Visit,0,
-316450,316405,VISIT,,,Intermediate Care Facility/Mentally Retarded,0,1,,8951,Visit,0,
-316451,316405,VISIT,,,Laboratory Visit,0,1,,32036,Visit,0,
-316452,316411,VISIT,,,Mass Immunization Center,0,1,,8858,Visit,0,
-316453,316405,VISIT,,,Military Treatment Facility,0,1,,8905,Visit,0,
-316454,316405,VISIT,,,Mobile Unit,0,1,,8584,Visit,0,
-316455,316405,VISIT,,,Non-residential Substance Abuse Treatment Facility,0,1,,8974,Visit,0,
-316456,316409,VISIT,,,Nursing Facility,0,1,,8676,Visit,0,
-316457,316405,VISIT,,,Observation Room,0,1,,581385,Visit,0,
-316458,316405,VISIT,,,Off Campus-Outpatient Hospital,0,1,,5084,Visit,0,
-316459,316411,VISIT,,,Office,0,1,,8940,Visit,0,
-316460,316405,VISIT,,,Office Visit,0,1,,581477,Visit,0,
-316461,316402,VISIT,,,Other Inpatient Care,0,1,,8892,Visit,0,
-316462,316403,VISIT,,,Outpatient Critical Care Facility,0,1,,581380,Visit,0,
-316463,316405,VISIT,,,Outpatient Critical Care Facility,0,1,,581380,Visit,0,
-316464,316405,VISIT,,,Outpatient Hospital,0,1,,8756,Visit,0,
-316465,316405,VISIT,,,Outpatient NEC,0,1,,8677,Visit,0,
-316466,316406,VISIT,,,Pharmacy,0,1,,8562,Visit,0,
-316467,316409,VISIT,,,Place of Employment-Worksite,0,1,,581475,Visit,0,
-316468,316405,VISIT,,,Prison/Correctional Facility,0,1,,38003619,Visit,0,
-316469,316405,VISIT,,,Psychiatric Facility-Partial Hospitalization,0,1,,8913,Visit,0,
-316470,316405,VISIT,,,Psychiatric Residential Treatment Center,0,1,,8976,Visit,0,
-316471,316411,VISIT,,,Public Health Clinic,0,1,,8977,Visit,0,
-316472,316409,VISIT,,,Residential Substance Abuse Treatment Facility,0,1,,8957,Visit,0,
-316473,316411,VISIT,,,Rural Health Clinic,0,1,,8761,Visit,0,
-316474,316409,VISIT,,,School,0,1,,8537,Visit,0,
-316475,316402,VISIT,,,Skilled Nursing Facility,0,1,,8863,Visit,0,
-316476,316409,VISIT,,,Temporary Lodging,0,1,,8602,Visit,0,
-316477,316405,VISIT,,,Tribal 638 Free-standing Facility,0,1,,8941,Visit,0,
-316478,316405,VISIT,,,Tribal 638 Provider-based Facility,0,1,,8960,Visit,0,
-316479,316411,VISIT,,,Urgent Care Facility,0,1,,8782,Visit,0,
-316480,316411,VISIT,,,Walk-in Retail Health Clinic,0,1,,38003620,Visit,0,
\ No newline at end of file
+"id","parent_id","type","subtype","code","name","is_group","is_selectable","est_count","domain_id","concept_id","has_attribute","predefined_attributes"
```

Comment:
What's the deal with all of the string quotations added?

Reply:
When generating these csv files with sequelPro it escapes everything with ". That way if any columns have special characters they don't mess up an import. Just to be on the safe side of things.

Reply:
Ok.